config/configModule.ts

```diff
@@ -0,0 +1,17 @@
+let env = process.env.NODE_ENV || 'development'
```

Comment:
The entire config system in the repo is just my boilerplate of how I juggle through tokens and environmental variables while deploying to Heroku. The config system needs to be updated entirely depending on how you want to deploy this.